### ISSUE TITLE
0.5.0 Clustered shots not working right for me

### ISSUE THREAD

**Issue by Epssus** [janxious/BT-WeaponRealizer#8]
I've installed WR 0.5.0, and a version of the LB 10-X (I created) seems to not work properly. I've properly tagged it as "wr-clustered_shots", but when it fires, it seems like I'm only getting a single hit instead of multiple cluster hits (the animation still properly plays multiple shots). Has something changed? The other functions of WR seem to working fine.

I'm using with mainly vanilla mods, not Roguetech or anything special.

**Comment by janxious** [comment]
Please provide your weapon json and modtek.log.

**Comment by Epssus** [comment]
Here are the two files. I've created a file using similar values to the one from Roguetech and I'm trying to get it to work in a "vanilla" mod.

Just to clarify what I'm seeing - on a mech with 2 autocannons equipped, I see only 1 or 2 damage numbers flash up when I get hits. I am operating under the assumption that with the projectiles set at 6 I should be seeing "up to" 12 numbers flash up. Is that right? (It also looks like the target Mech isn’t taking much damage)


[Weapon_Autocannon_LB10-X_CLUSTER_0-STOCK.zip](https://github.com/janxious/BT-WeaponRealizer/files/2783984/Weapon_Autocannon_LB10-X_CLUSTER_0-STOCK.zip)

**Comment by Epssus** [comment]
Here's a followup - I finally got a chance to fire off a pair of the LB 10-X I am trying to get to work, and also capture it in output_log.txt, and here's what I got:

`(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

CombatLog.MechImpacts [ERROR] MessageCoordinator errors detected!
ExpectedMessage 0 ExpectedImpact Completed = True, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = True, TimesClearedForProcessing = 1
	Group = 0, Weapon = 0, Shot = 0
ExpectedMessage 1 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0, Shot = 1
ExpectedMessage 2 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0, Shot = 2
ExpectedMessage 3 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0, Shot = 3
ExpectedMessage 4 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0, Shot = 4
ExpectedMessage 5 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0, Shot = 5
ExpectedMessage 6 ExpectedResolveDamange Completed = False, StoredEarlyMessage = True, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 0
ExpectedMessage 7 ExpectedImpact Completed = False, StoredEarlyMessage = True, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 1, Shot = 0
ExpectedMessage 8 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 1, Shot = 2
ExpectedMessage 9 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 1, Shot = 3
ExpectedMessage 10 ExpectedImpact Completed = False, StoredEarlyMessage = False, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 1, Shot = 5
ExpectedMessage 11 ExpectedResolveDamange Completed = False, StoredEarlyMessage = True, SentEarlyMessage = False, ClearedForProcessing = False, TimesClearedForProcessing = 0
	Group = 0, Weapon = 1`

The second shot missed. I'm not sure what happened to messages for shots 1 and 4 on the second volley.
